indexer/services/vulcan/src/handlers/order-place-handler.ts

```diff
@@ -150,6 +151,7 @@ export class OrderPlaceHandler extends Handler {
           dbOrder,
           perpetualMarket,
           placementStatus,
+          blockHeightRefresher.getLatestBlockHeight(),
```

Comment:
Is this expected to be the block that this order belonged to? Can there be a situation where the latest block height is increased by ender before this is called such that this is a later block than the one corresponding to this order?

Reply:
No, this is the latest block processed by Ender. Yes, that situation is possible.